tests/org.eclipse.n4js.xpect.tests/xt-tests/typesystemPT/variableFieldTypeInference/Importee.n4js

```diff
@@ -18,8 +18,8 @@ export public class Remote {
 	public fieldFN_decl : {function(string):number} = function(p : string) : number { return 0; };
 	public                           fieldFN_inf  = function(p : string) : number { return 0; };
 	
-	public fieldOL_decl : ~Object with { s: string; n: number; } = { s: "hello", n: 42.0 };
-	public                                      fieldOL_inf  = { s: "hello", n: 42.0 };
+	public fieldOL_decl : ~Object with { s: string; n: number; } = { s: "hello", n: 42.3 };
+	public                                      fieldOL_inf  = { s: "hello", n: 42.3 };
```

Comment:
I don't remember but did you need to change these digits because otherwise this literal would be interpreted as a whole number? If so, at least in Java, there are some situations where it is handy to define floats adding `.0`, right?

Reply:
Good point. At some point I thought it's ok to infer `1.0` to "int" instead of "number" and therefore changed these tests. A few days later I thought better of it (now `1.0` is inferred to "number") but forgot to revert the changes in this file (and 2 others).

Done!